src/main/java/frc/robot/TestMode.java

```diff
@@ -142,4 +154,18 @@ private Command testArmCommand() {
         .andThen(() -> armPublisher.set(armSubsystem.isArmAtAngle()))
         .finallyDo(armSubsystem::park);
   }
+
+  /**
+   * Returns the results of the tests ran during the TestMode routine
+   * 
+   * @return test results
+   */
+  public BooleanSupplier[] getTestResults() {
+    BooleanSupplier[] results = new BooleanSupplier[testResults.length];
+    for (int i = 0; i < testResults.length; i++) {
+      final int index = i;
```

Comment:
This line isn't really needed, you could just use `i`.

I'm also a big fan of streams. This whole method could be done pretty simply:
```
Arrays.stream(testResults).map(BooleanSupplier::getAsBoolean).toArray();
```